linkage/pvs_like_case_study/generate_simulated_data/generate_simulated_data_small_sample.ipynb

```diff
@@ -9,7 +9,8 @@
    "source": [
```

Comment:
<p>I bet our Census Bureau colleagues can tell us <em>something </em>about how we can improve our address schema.  Let's ask.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>